share/commands/agent-run

```diff
@@ -60,14 +60,17 @@ while getopts "iIvdqwrRmcb:D:" opt; do
     i|I)
       VERBOSITY="-I ${DEBUG_CLASS}"
       DISPLAY_INFO=1
+      QUIET=0
```

Comment:
Avoid strange behavior when passing several verbosity level, the last one only should be applied.